pkg/standalone/standalone.go

```diff
@@ -1065,14 +1086,87 @@ func downloadFile(dir string, url string) (string, error) {
 	}
 	defer out.Close()
 
-	_, err = io.Copy(out, resp.Body)
+	_, err = copyWithTimeout(context.Background(), out, resp.Body)
 	if err != nil {
 		return "", err
 	}
 
 	return filepath, nil
 }
 
+//see https://github.com/microsoft/vscode-winsta11er/blob/main/common/common.go#L169
```

Comment:
We need to add the copyright notice here to be compliant with the MIT license. Please change this to:

```go
/*!
See: https://github.com/microsoft/vscode-winsta11er/blob/4b42060da64aea6f47adebe1dd654980ed87a046/common/common.go
Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License.
*/
```